manifests/init.pp

```diff
@@ -141,50 +142,69 @@
   Enum['RAW','ENRICHED','NOLOG']          $log_format              = 'RAW',
   String                                  $log_group               = 'root',
-  Integer[0]                              $priority_boost          = 3,
-  Auditd::Flush                           $flush                   = 'INCREMENTAL',
-  Integer[0]                              $freq                    = 20,
-  Integer[0]                              $num_logs                = 5, # CCE-27522-2
-  Enum['lossy','lossless']                $disp_qos                = 'lossy',
-  Stdlib::Absolutepath                    $dispatcher              = '/sbin/audispd',
-  Auditd::NameFormat                      $name_format             = 'USER',
+  String                                  $lname                   = $facts['fqdn'],
   Integer[0]                              $max_log_file            = 24, # CCE-27550-3
   Auditd::MaxLogFileAction                $max_log_file_action     = 'ROTATE', # CCE-27237-7
-  Integer[0]                              $admin_space_left        = 50,
-  Auditd::SpaceLeftAction                 $admin_space_left_action = 'SUSPEND', # CCE-27239-3 : No guarantee of e-mail server so sending to syslog.
+  Optional[Integer[1]]                    $max_restarts            = undef,           #data            = 10, #auditd version 3.0 and later
+  Auditd::NameFormat                      $name_format             = 'USER',
+  Integer[0]                              $num_logs                = 5, # CCE-27522-2
+  Optional[Auditd::Overflowaction]        $overflow_action          = undef,         # data in module
+  String[1]                               $package_name            = 'audit',
+  Simplib::PackageEnsure                  $package_ensure          = simplib::lookup('simp_options::package_ensure', { 'default_value' => 'installed' }),
+  Stdlib::Absolutepath                    $plugin_dir,             # data in module
+  Integer[0]                              $priority_boost          = 3,
+  Integer[0]                              $q_depth                 = 400,
+  Integer[0]                              $rate                    = 0,
+  Auditd::RootAuditLevel                  $root_audit_level        = 'basic',
+  String[1]                               $service_name            = 'auditd',
   Integer[0]                              $space_left              = $admin_space_left + 25, # needs to be larger than $admin_space_left or auditd will not start
   Auditd::SpaceLeftAction                 $space_left_action       = 'SYSLOG', # CCE-27238-5 : No guarantee of e-mail server so sending to syslog.
-  String[1]                               $action_mail_acct        = 'root', # CCE-27241-9
-  Auditd::DiskFullAction                  $disk_full_action        = 'SUSPEND',
-  Auditd::DiskErrorAction                 $disk_error_action       = 'SUSPEND',
+  Boolean                                 $manage_syslog_plugin    = false,   # CCE-26933-2
```

Comment:
Not linked to simp_options::syslog anymore.  Is that what we want?

Reply:
It is what we want but for backwards compatibility, so people's settings dont' change, I put it back.  (Because we aren't doing a major version change any longer.)  You actually want people to chose whether or not they want to include the plugin.  It is not tied to wheher you have syslog set up or not.  You can have syslog configured and still not want your auditd logs going to syslog.